source/test/eslint-compatibility.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from 'path';
```

Comment:
```suggestion
import path from 'path';
```

Reply:
This won't work because `esModuleInterop` isn't enabled in this project. Should I enable it and refactor it?

Reply:
> Should I enable it and refactor it?

👍

Reply:
done